pkg/config/autodiscover/csi_info.go

```diff
@@ -0,0 +1,58 @@
+package autodiscover
+
+import (
+	"fmt"
+	"strings"
+)
+
+// PodSetList holds the data from an `oc get deployment/statefulset -o json` command
+
+// PodSetResource defines deployment/statefulset resources
+var (
+	csicommand          = "oc get csidriver -o go-template='{{ range .items}}{{.metadata.name}} {{end}}'"
+	depnamecpmmand      = "oc get pods -A -o go-template='{{ range .items}}{{ $alllabels := .metadata.labels}}{{ $namespace := .metadata.namespace}}{{ range .spec.containers }}{{ range .args }}{{if eq . \"--driver-name=%s\"}}{{ range $label,$value := $alllabels}}{{if eq $label \"app.kubernetes.io/managed-by\"}}{{$value}} {{$namespace}}{{end}}{{end}}{{end}}{{end}}{{end}}{{end}}'"
+	operatornamecommand = "oc get deployment %s -n %s -o go-template='{{ range $label,$value := .metadata.labels}}{{$label}}{{print \"\n\"}}{{end}}' |grep \"operators.coreos.com\"| sed \"s#operators.coreos.com/##g\""
+	subscriptioncommand = "oc get operator %s -o go-template='{{ range .status.components.refs }}{{if eq .kind \"Subscription\"}}{{.name}}{{end}}{{end}}'"
+	orgpackcommand      = "oc get subscription -n %s %s -o go-template='{{.spec.source}} {{.spec.name}}'"
+)
+
+func getpackageandorg(csi string) (packag, organization string) {
```

Comment:
Please use upper case in variable and function names as golang standard is camel case